src/components/adaptive-cards/InputToggle/InputToggle.jsx

```diff
@@ -8,16 +10,62 @@ import {acPropTypes, registerComponent} from '../Component/Component';
  *
  * @param {object} props  React props passed to the component
  * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Inline style for the component
  * @returns {object} JSX of the component
  */
-export default function InputToggle({data}) {
+export default function InputToggle({data, className, style}) {
+  const [cssClasses, sc] = webexComponentClasses('adaptive-cards-input-toggle', className);
+  const {value = 'false', valueOn = 'true', valueOff = 'false'} = data;
+  const {getValue, setInput, setValue} = useContext(AdaptiveCardContext);
+  const inputValue = getValue(data.id);
```

Comment:
for consistency with other components, let's use the name `value` for the current value of the input (got from getValue()).

data.value can be called initialValue or can remain as data.value, as you prefer